ncm-systemd/src/main/resources/unitfile/section.tt

```diff
@@ -11,6 +11,6 @@
 [%-          CASE long_infinity -%]
 [%-              pair.key %]=[% value == -1 ? 'infinity' : value %]
 [%           CASE -%]
-[%-              pair.key %]=[% INCLUDE 'systemd/unitfile/attribute_value.tt' data=value %]
+[%-              pair.key %]=[% INCLUDE 'systemd/unitfile/attribute_value.tt' data=value octals=['CacheDirectoryMode', 'ConfigurationDirectoryMode', 'LogsDirectoryMode', 'RuntimeDirectoryMode', 'StateDirectoryMode'] %]
```

Comment:
split the line; if all keys ending with `Mode` are octals, best to match this in the CASE statements instead of maintaining the list

Reply:
As far as I currently understand, you have to write it in base 10 and this will convert it to octal right?
So we have to write `mask = 511` to get `777` ? Isn't writing it as a string a bit more readable?

Reply:
what do you mean "write in base10"? in pan, you can also write `mask = 0777`